.travis.yml

```diff
@@ -1,7 +1,7 @@
-sudo: false
+language: scala
 cache:
   directories:
-    - '$HOME/.m2/repository'
+    - '$HOME/node_modules'
```

Comment:
I tried that and it didn't seem to help, but there's probably something we can do to cache the dependencies. Although, I'm not sure it matters all that much as this build doesn't run all that often.